_sass/_layouts--global.scss

```diff
@@ -11,3 +11,12 @@ body {
     flex: 1;
   }
 }
+
+body {
+  // left the gap for header navbar
+  padding-top: $size--header-height;
```

Comment:
Not so sure about this, as most pages seen to not need it, but we can add `pt-0` class from bootstrap on those pages